src/Platform.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Utopia\Platform;
+
+use Utopia\App;
+
+abstract class Platform {
+    protected array $services = [];
+
+    public function init():void
+    {
+        foreach ($this->services as $key => $service) {
+            if($service->getType() === Service::TYPE_HTTP) {
```

Comment:
We can use a switch case and have a dedicated private method for each init. For example `initHTTP` and `initGraphQL`